### ISSUE TITLE
Messages in console always visible

### ISSUE THREAD

**Issue by jalamprea** [questdb/nodejs-questdb-client#12]
I want to use this module, but since it uses console.info to show some messages, it is a little bit slow due to the sync code with console.
The debug/info messages should be optional and defined as part of the options when the user create the sender

**Comment by glasstiger** [comment]
The client prints `info` level messages only in `connect()`/`close()` and that is intentional.
Expected the client app to connect to the DB at startup and then hold onto the client until shutdown or until it is not needed anymore.
Maybe you create a new client often?

The plan was (and I think still is) to give the option to inject a logger into the client so it can integrate with the host app's logging system.
At that point even `debug` level logging could be added, client could print entire traffic sent to QuestDB if `debug` level logging is enabled.

**Comment by glasstiger** [comment]
Probably will merge your pull request for now to get you going and will look into logger integration after that.

**Comment by glasstiger** [comment]
Just released v1.0.2 which has a new option to pass a custom logging function to the client.
This should help.
Now you can create a Sender which does not log anything:
```
const sender = new Sender({bufferSize: 1024, log: () => {}});
```

**Comment by glasstiger** [comment]
@jalamprea, closing this issue now.
Please, feel free to reopen it if the suggested solution is not satisfactory.
